src/main/content/_includes/site-translation.html

```diff
@@ -0,0 +1,53 @@
+<script type="text/javascript"
+  src="https://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
+<script>
+  function googleTranslateElementInit() {
+    new google.translate.TranslateElement({ pageLanguage: 'en', layout: google.translate.TranslateElement.FloatPosition.TOP_LEFT }, 'google_translate_element');
+  }
+  let currentLanguage = `{{page.language}}`
+  let selectedLanguage = navigator.languages && navigator.languages[0] || navigator.language || navigator.userLanguage;
+  if (currentLanguage !== selectedLanguage) {
+    if (selectedLanguage === 'en' || selectedLanguage === 'ru' || selectedLanguage === 'es' || selectedLanguage === 'de' || selectedLanguage === 'fr' || selectedLanguage === 'it' || selectedLanguage === 'ja' || selectedLanguage === 'ko' || selectedLanguage === 'pt-BR' || selectedLanguage === 'zh-CN' || selectedLanguage === 'zh-TW') {
```

Comment:
don't need this if statement, the switch case should handle all the cases and provide an english default